config-editor/config-editor-ui/src/app/services/clipboard.service.ts

```diff
@@ -0,0 +1,73 @@
+import { Injectable } from '@angular/core';
+import { ConfigData } from '@app/model';
+import { from, Observable, throwError } from 'rxjs';
+import { EditorService } from './editor.service';
+
+@Injectable({
```

Comment:
try to generate in EditorService context during initialization of the service context. Try to use BehavioralSubject similarly as we use in config store to store and update values as observable.